app/views/topics/new.html.erb

```diff
@@ -1,8 +1,22 @@
 <h2>Topics</h2>
 <p>Add a new topic</p>
 
-<%= form_for @topic, url: {action: "create"}, html: {class: "nifty_form"} do |f| %>
-  <%= f.label :name %>
-  <%= f.text_field :name, :required => 'required' %>
-  <%= f.submit "Create" %>
+<div>
+  <%= form_for @topic do |f| %>
+    <%= f.label :name %>
+    <%= f.text_field :name, id: :name %>
+    <%= f.submit %>
+  <% end %>
+</div>
+
+
+<% if @topic.errors.any? %>
+  <div>
+    <h3>Error:</h3>
```

Comment:
If the heading on this page is an `<h1>` then this should be a `<h2>`. 
(I notice the heading for this page is not a h1. Tut tut 😜 )